spec/system/admin/sdg/managers_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "rails_helper"
+
+describe "Admin SDG managers" do
+  let!(:user) { create(:user) }
+  let!(:sdg_manager) { create(:sdg_manager) }
+
+  before do
+    login_as(create(:administrator).user)
+    visit admin_sdg_managers_path
+  end
+
+  scenario "Index" do
+    expect(page).to have_content sdg_manager.name
+    expect(page).to have_content sdg_manager.email
+    expect(page).not_to have_content user.name
+  end
+
+  scenario "Create SDG Manager", :js do
```

Comment:
If possible, use the `:js` tag in the first `describe`, so it's used in the whole file :wiggum: :smile:.